templates/taxi/manufacturer_list.html

```diff
@@ -2,6 +2,7 @@
 
 {% block content %}
     <h1>Manufacturer List
+    <a class="float-right btn btn-primary" href="{% url "taxi:manufacturer-create" %}">Create</a>
```

Comment:
Consider using single quotes in the `{% url %}` template tag: `{% url 'taxi:manufacturer-create' %}`. This is a recommended practice to avoid conflicts with HTML attributes that use double quotes.